src/components/calculator.js

```diff
@@ -1,11 +1,54 @@
-/* eslint-disable react/button-has-type */
 import { PropTypes } from 'prop-types';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import calculate from '../logic/calculate';
 
-// let counter = 0;
 const ans = document.querySelector('.answer');
 
+const Quotes = () => {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=learning', {
+          headers: {
+            'X-Api-Key': 'baoB69sSSBvXDFb2gkXx6w==5nrM0rh00PkzY7G0',
+          },
+        });
+        const json = await res.json();
+        setData(json);
+      } catch (error) {
+        setHasError(true);
+      }
```

Comment:
- [ ] As requested earlier, I think it would be better to create a separate `quote.js` file responsible for fetching quotes data.  By creating separate components, you can scale your application more easily. As your application grows, you can add more components to it without having to worry about how they will interact with other parts of your code.

Reply:
Created the new file and made adjustments as per instructions